File: CPP_Joc_Windows_Android/SH_Client_Win_Cpp_Cmake/App/src/base/menu/util/MenuItemConfig_ComposedVisual.cpp
#include "MenuItemConfig_ComposedVisual.h"

MenuItemConfig_ComposedVisual::MenuItemConfig_ComposedVisual(std::string itemID, ArrayList<MenuItemConfig*>* menuItemConfigs)
    : super(itemID), menuItemConfigs(menuItemConfigs)
{
    //void
}

ArrayList<MenuItemConfig*>* MenuItemConfig_ComposedVisual::getMenuItemConfigs() {
    return menuItemConfigs;
}

MenuItemConfig_ComposedVisual::~MenuItemConfig_ComposedVisual() {
    //void
}

File: Java_Joc_Windows_Android/AndroidStudioProject/core/src/base/menu/util/MenuItemConfig_ComposedVisual.java
package base.menu.util;

public class MenuItemConfig_ComposedVisual extends MenuItemConfig implements IMenuItemConfig_ComposedVisual {

	private MenuItemConfig[] menuItemConfigs;

	public MenuItemConfig_ComposedVisual(String itemID, MenuItemConfig[] menuItemConfigs) {
		super(itemID);

		this.menuItemConfigs = menuItemConfigs;
	}

	@Override
	public MenuItemConfig[] getMenuItemConfigs() {
		return menuItemConfigs;
	}

}
